tbx/project_styleguide/templates/patterns/pages/work/work_page.html

```diff
@@ -75,7 +20,7 @@ <h1 class="heading heading--one">{{ page.title }}</h1>
             <picture>
                 <source media="(max-width: 598px)" srcset="{{ mobile_header_image.url }} 1x, {{ mobile_header_image_retina.url }} 2x" />
                 <source media="(min-width: 599px)" srcset="{{ desktop_header_image.url }} 1x, {{ desktop_header_image_retina.url }} 2x" />
-                <img src="{{ desktop_header_image.url }}" alt="{{ desktop_header_image.alt }}" loading="lazy" />
+                <img src="{{ desktop_header_image.url }}" alt="{{ desktop_header_image.alt }}" loading="lazy" class="image" />
```

Comment:
Just checking that adding the `image` class here is intentional as the same class is used on line 14:
`<div class="grid__work-image image">`

Reply:
Yep that was an oversight - removed.